rules-tests/Transform/Rector/Class_/RemoveAllowDynamicPropertiesAttributeRector/UnconfiguredRemoveAllowDynamicPropertiesAttributeRectorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\Transform\Rector\Class_\RemoveAllowDynamicPropertiesAttributeRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+final class UnconfiguredRemoveAllowDynamicPropertiesAttributeRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        $this->doTestFileInfo($fileInfo);
+    }
+
+    /**
+     * @return Iterator<SmartFileInfo>
+     */
+    public function provideData(): Iterator
+    {
+        return $this->yieldFilesFromDirectory(__DIR__ . '/Fixture/Process');
```

Comment:
you can add new Fixture with different name, eg: `FixtureAllClasses`

Reply:
Oh - should the fixtures used for the unconfigured rule be different? I'm not sure I understand completely. Is there an existing rule (and matching tests) that follows a similar convention for me to refer to?

I thought it would make sense to re-use the same test fixtures as the configured one. However, the configured one must include at least one rule for it to exclude due not matching to the namespace includes.

Is the suggestion to create a new `FixtureAllClasses` along side the existing `Fixture` folder, duplicate the `Fixture\Process\*` files into that and let the Unconfigured version use that instead?

Reply:
there are multiple `Fixture*` directory example at https://github.com/rectorphp/rector-src/tree/main/rules-tests/Php74/Rector/Property/TypedPropertyRector

eg:

```php
FixtureClassLikeTypeOnly
FixtureDoctrine
```

For all classes, I think namespace check should be skipped, so even class has no namespace, it will always applied.

Reply:
I think I follow now - added that folder name and fixtures that don't include any namespaces to be checked.